res/pokemon/abomasnow/data.json

```diff
@@ -136,7 +136,34 @@
         "pokemon_pos_f": 4,
         "trainer_pos_m": 15,
         "pokemon_pos_m": 4,
-        "entry_text": "They appear when the snow flowers\nbloom. When the petals fall, they\nretreat to places unknown again.",
-        "category": "Frost Tree Pokémon"
+        "english": {
+            "category": "Frost Tree Pokémon",
+            "entry_text": "They appear when the snow flowers\nbloom. When the petals fall, they\nretreat to places unknown again."
+        },
+        "french": {
+            "name": "BLIZZAROI",
+            "category": "Pokémon Arbregelé",
+            "entry_text": "C’est l’abominable homme des\nneiges. Il ensevelit les montagnes\nsous ses blizzards."
+        },
+        "german": {
+            "name": "REXBLISAR",
+            "category": "Frostbaum",
+            "entry_text": "Es löst in den Bergen, wo ewiger\nSchnee liegt, Blizzards aus. Es ist\nein scheußlicher Schneemann."
+        },
+        "italian": {
+            "name": "Abomasnow",
+            "category": "Pokémon Albergelo",
+            "entry_text": "Crea bufere di neve sui monti\nricoperti da nevi perenni.\nÈ l’abominevole essere delle nevi."
+        },
+        "spanish": {
+            "name": "Abomasnow",
+            "category": "Pokémon Árbol Nieve",
+            "entry_text": "Es el abominable hombre de las\nnieves. Produce ventiscas en las\nmontañas cubiertas de nieve."
+        },
+        "japanese": {
+            "name": "ユキノオ－",
+            "category": "じゅひょうポケモン",
+            "entry_text": "まんねんゆきの　つもる　やまで\nくらし　ブリザ－ドを　まきおこす。\nゆきおとこの　しょうたいだ。"
+        }
```

Comment:
**suggestion**: I think that we should consider moving Pokedex entries to a data-markup format that permits multi-line strings. These raw `\n` characters are quite ugly, in my opinion.

Reply:
I will look into yaml. Should probably move all Pokedex data into this separate file.